detail.scss

```diff
@@ -0,0 +1,72 @@
+$background-color:#FFF;
+$border-color:#BED23C;
+$main-font:"Roboto";
```

Comment:
Move this ones into a `variables.scss` file

Reply:
Done